src/robot/robot.ts

```diff
@@ -146,4 +148,15 @@ export interface Robot {
     resourceNames?: ResourceName.AsObject[],
     durationMs?: number
   ): RobotStatusStream;
+
+  /**
+   * Call a function when an event of either 'reconnected' or 'disconnected' is 
+   * triggered.
+   * 
+   * @param event - The event that was triggered.
+   * @param listener - The function to call
+   * @group Connection
+   * @alpha
```

Comment:
What `@group` should this be in, if any? What does `@alpha` do?

Reply:
the group is purely for organization -  we can change this later or even leave it off. `Event Listeners` might be a good name if we end up adding non-connection events later.